File: api-server/src/main/java/com/talentlink/talentlink/chat/dto/ChatMessageDto.java
package com.talentlink.talentlink.chat.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
public class ChatMessageDto {

    private Long id;
    private Long roomId;
    private Long senderId;
    private String senderNickname;
    private String content;
    private LocalDateTime sentAt;
    private boolean isRead;

}

File: front-server/app/src/main/java/com/example/talent_link/ui/chat/dto/ChatMessageDto.kt
package com.example.talent_link.Chat.dto

import com.google.gson.annotations.SerializedName

data class ChatMessageDto(
    val id: Long? = null,
    val roomId: Long,
    val senderId: Long,
    val senderNickname: String,
    val content: String,
    val sentAt: String? = null,  // ISO 형식 시간 문자열 (서버 반환)
    @SerializedName("read")
    val isRead: Boolean = false
)
